src/main/java/io/seoul/helper/controller/team/TeamApiController.java

```diff
@@ -0,0 +1,28 @@
+package io.seoul.helper.controller.team;
+
+import io.seoul.helper.config.auth.LoginUser;
+import io.seoul.helper.config.auth.dto.SessionUser;
+import io.seoul.helper.controller.team.dto.TeamCreateRequestDto;
+import io.seoul.helper.service.TeamService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@Slf4j
+@RestController
+public class TeamApiController {
+    @Autowired
+    TeamService teamService;
+
+    @PostMapping(value = "/api/v1/create_team")
```

Comment:
```suggestion
    @PostMapping(value = "/api/v1/teams") // HTTP REST API 
```
또는 
```suggestion
    @PostMapping(value = "/api/v1/teams/new") //html form 전송시
```
를 제안드립니다. 

아직 저희가 API 설계를 진행하지 않았기 때문에 참고 코멘트라고 생각하시면 됩니다.
RestController의 행위에 대해서 직관적으로 잘 작성해주셨다고 생각합니다.

회의에서 잠깐 이야기를 해보고자 comment 를 달았습니다.

Reply:
https://okdevtv.com/mib/restapi
@epicarts 잘 하셨습니다.